### PR TITLE
Configuring attestation service on Alfajores to use the Telekom provider

### LINKED ISSUE #6112: Roll-out AS Release 1.2.0
Publish latest version following Release Process:
https://docs.celo.org/community/release-process/attestation-service

### PR DESCRIPTION
**Pull request by AlexBHarley**
### Description

Allow passing additional options to Attestation Service and configure Alfajores to use Telekom. 

### Tested

Deployed Attestation Service with these variables.

### Related issues

- Fixes https://github.com/celo-org/celo-monorepo/issues/6112

### CODE REVIEW

File: .env.alfajores [hunk 1]
```diff
@@ -78,9 +78,21 @@ EKSPORTISTO_DOCKER_IMAGE_REPOSITORY="us.gcr.io/celo-testnet/eksportisto"
...
+ATTESTATION_BOT_MAX_ATTESTATIONS=32
+
+# If this is set, the bot will skip ODIS and use this salt (needed for baklava, where ODIS is not deployed)
+ATTESTATION_BOT_SKIP_ODIS_SALT=bot
+
 ATTESTATION_SERVICE_DOCKER_IMAGE_REPOSITORY="us.gcr.io/celo-testnet/celo-monorepo"
 ATTESTATION_SERVICE_DOCKER_IMAGE_TAG="attestation-service-v1.2.0"
 
+TELEKOM_FROM="+14157389085"
+SMS_PROVIDERS=twilio,nexmo,telekom
```
Comment (timmoreton) [inline .env.alfajores:93]: could also add MessageBird here

File: .env.baklava [hunk 2]
```diff
@@ -114,6 +114,9 @@ EKSPORTISTO_SUFFIX='1'
 ATTESTATION_SERVICE_DOCKER_IMAGE_REPOSITORY="us.gcr.io/celo-testnet/celo-monorepo"
 ATTESTATION_SERVICE_DOCKER_IMAGE_TAG="attestation-service-v1.2.0"
 
+SMS_PROVIDERS=twilio,nexmo
```
Comment (timmoreton) [inline .env.baklava:120]: could also add MessageBird here

File: .env.alfajores (hunk 1 above)
Comment (timmoreton) [reply .env.alfajores:93]: (or else just rely on telekom?)

File: packages/helm-charts/attestation-bot/templates/secret.yaml [hunk 3]
```diff
@@ -12,4 +12,5 @@ type: Opaque
 data:
   mnemonic: {{ .Values.mnemonic | b64enc }}
   twilioAccountSid: {{ .Values.twilio.accountSid | b64enc }}
+  twilioAddressSid: {{ .Values.twilio.addressSid | b64enc }}
```
Comment (timmoreton) [inline packages/helm-charts/attestation-bot/templates/secret.yaml]: is this actually used by the bot any more?

File: packages/helm-charts/attestation-bot/templates/statefulset.yaml [hunk 4]
```diff
@@ -56,6 +56,11 @@ spec:
             secretKeyRef:
               name: {{ .Values.environment }}-attestation-bot-secrets
               key: twilioAccountSid
+        - name: TWILIO_ADDRESS_SID
```
Comment (timmoreton) [inline packages/helm-charts/attestation-bot/templates/statefulset.yaml]: ditto

File: packages/helm-charts/attestation-bot/templates/secret.yaml (hunk 3 above)
Comment (AlexBHarley) [reply packages/helm-charts/attestation-bot/templates/secret.yaml:15]: https://github.com/celo-org/celo-monorepo/blob/master/packages/env-tests/src/tests/attestation.ts#L80

File: .env.alfajores (hunk 1 above)
Comment (AlexBHarley) [reply .env.alfajores:93]: Prefer to get this in as its been open for a while and add message bird in a separate PR

File: .env.baklava (hunk 2 above)
Comment (AlexBHarley) [reply .env.baklava:120]: Prefer to get this in as its been open for a while and add message bird in a separate PR

**Review by aslawson — APPROVED** [review]

**Pull request merged**